Frontend/ui-driver/src/RemoteConfigs/Utils.purs

```diff
@@ -174,6 +174,11 @@ defaultMetroCoinsEvent = {
   minDistance : 0
 }
 
+getParcelConfig :: String -> ParcelConfig
+getParcelConfig lazy = 
+  let config = fetchRemoteConfigString "parcel_config"
+  in decodeForeignObject (parseJSON config) $ {introductionVideo : "https://www.youtube.com/playlist?list=PLvMgI4c44A9bZPI3XouAG6MSjG85QXjS3"}
```

Comment:
@max-keviv should we add the default case video in v1-config such that this can be updated later on if required?

Reply:
I guess remote config can fulfill the dynamic requirement based on cities if it comes